### ISSUE TITLE
Editor crash

### ISSUE THREAD

**Issue by mllev** [paladin-t/bitty#39]
When playing with the included example, Physics/Sticky, the editor crashes with the following error when I click on one of the sticky objects, and then I drag my mouse off screen to the right. I'm using x64/Bitty on Pop!_OS.

```
Code editor opened: "main.lua".
Initializing cpSpace - Chipmunk v7.0.3 (Debug Enabled)
Compile with -DNDEBUG defined to disable debug mode and runtime assertion checks
Aborting due to Chipmunk error: Unsolvable constraint.
	Failed condition: det != 0.0
	Source:/home/tony/Documents/shared/bitty/bitty.linux/../lib/chipmunk2d/include/chipmunk/chipmunk_private.h:255
Aborted (core dumped)
```

**Comment by paladin-t** [comment]
I would like to know more specific, which distribution is it, itch.io or steam?

**Comment by paladin-t** [comment]
Fixed in v1.2.2.
